frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -31,15 +31,20 @@ common_settings.network = {
     text = _("Network"),
     sub_item_table = NetworkMgr:getMenuTable()
 }
+
 common_settings.screen = {
     text = _("Screen"),
     sub_item_table = {
         require("ui/elements/screen_dpi_menu_table"),
         require("ui/elements/screen_eink_opt_menu_table"),
         require("ui/elements/screen_disable_double_tap_table"),
-        require("ui/elements/refresh_menu_table"),
+        require("ui/elements/refresh_menu_table")
     },
 }
+if Device:isAndroid() then
+    table.insert(common_settings.screen.sub_item_table, require("ui/elements/screen_fullscreen_menu_table"))
```

Comment:
Could you be a little more specific about "a problem"? The reference to `common_settings.screen.sub_item_table` seems fine in a quick test.